### ISSUE TITLE
False Positives for org.mongodb.morphia

### ISSUE THREAD

**Issue by albuch** [dependency-check/DependencyCheck#964]
False positive on libraries morphia-1.3.2.jar, morphia-logging-1.3.2.jar, morphia-logging-slf4j-1.3.2.jar - reported as `cpe:/a:git:git` and `cpe:/a:git_project:git`
```xml
<dependency>
   <groupId>org.mongodb.morphia</groupId>
   <artifactId>morphia</artifactId>
   <version>1.3.2</version>
</dependency>
```
It seems the Git-URL is used as evidence.
We've used the following suppression rule
```
    <suppress>
        <notes><![CDATA[
   file name: morphia-logging-slf4j-1.3.2.jar
   ]]></notes>
        <gav regex="true">^org\.mongodb\.morphia:.*$</gav>
        <cpe>cpe:/a:git:git</cpe>
        <cpe>cpe:/a:git_project:git</cpe>
    </suppress>
```

**Comment by jeremylong** [comment]
Thanks!  I'll do a sprint cleaning up all the report FP before the next release.
